mteb/tasks/PairClassification/zho/CMTEBPairClassification.py

```diff
@@ -64,48 +68,52 @@ class Cmnli(AbsTaskPairClassification):
         dialect=None,
         text_creation=None,
         bibtex_citation="""@inproceedings{xu-etal-2020-clue,
-    title = "{CLUE}: A {C}hinese Language Understanding Evaluation Benchmark",
-    author = "Xu, Liang  and
-      Hu, Hai  and
-      Zhang, Xuanwei  and
-      Li, Lu  and
-      Cao, Chenjie  and
-      Li, Yudong  and
-      Xu, Yechen  and
-      Sun, Kai  and
-      Yu, Dian  and
-      Yu, Cong  and
-      Tian, Yin  and
-      Dong, Qianqian  and
-      Liu, Weitang  and
-      Shi, Bo  and
-      Cui, Yiming  and
-      Li, Junyi  and
-      Zeng, Jun  and
-      Wang, Rongzhao  and
-      Xie, Weijian  and
-      Li, Yanting  and
-      Patterson, Yina  and
-      Tian, Zuoyu  and
-      Zhang, Yiwen  and
-      Zhou, He  and
-      Liu, Shaoweihua  and
-      Zhao, Zhe  and
-      Zhao, Qipeng  and
-      Yue, Cong  and
-      Zhang, Xinrui  and
-      Yang, Zhengliang  and
-      Richardson, Kyle  and
-      Lan, Zhenzhong",
-    booktitle = "Proceedings of the 28th International Conference on Computational Linguistics",
-    month = dec,
-    year = "2020",
-    address = "Barcelona, Spain (Online)",
-    publisher = "International Committee on Computational Linguistics",
-    url = "https://aclanthology.org/2020.coling-main.419",
-    doi = "10.18653/v1/2020.coling-main.419",
-    pages = "4762--4772",
-}""",
+            title = "{CLUE}: A {C}hinese Language Understanding Evaluation Benchmark",
+            author = "Xu, Liang  and
+            Hu, Hai  and
+            Zhang, Xuanwei  and
+            Li, Lu  and
+            Cao, Chenjie  and
+            Li, Yudong  and
+            Xu, Yechen  and
```

Comment:
I don't think we should change the bibtex in any of these cases.

Reply:
Well, `ruff` doens't care, and we're not consisnent with it over the datasets. So I admit, it is quite opinionated, but imho lack of identation looks weird and confusing. But if you feel stronlgy about it I can bring it back to what it was...

Reply:
Right - @Sakshamrzt, just checking if this affects you?

If it does not I am perfectly fine keeping it as you suggest

Reply:
Let us assume it is fine for now (I think it is)